### PR TITLE
[i18n-KO] Translated chapter 2-1 page to Korean

### PR DESCRIPTION
**Pull request by wonhyeongseo**
# What does this PR do?

Translated the `1.mdx` file of chapter 2 for the course to Korean. I use transformers for my side-project and want to contribute. As a beginner, this seemed appropriate.

Some questions:
- Package names and their everyday term counterpart is quite confusing to differentiate. Is there a way to tell easily? (🤗 transformers and just the model, 🤗 tokenizers and just regular tokenizers etc.)
- I also got to know a fellowship for huggingface. How do I apply? Do I need a nomination from a prior fellow?
- I checked the previous assignee's repo before this PR. What is the point of no return? 3 months since the last commit? When can another person takeover? (still very new to opensource, sorry for the novice question)

Thank you in advance for your review. 

Part of https://github.com/huggingface/course/issues/42

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?
@sgugger, @lewtun, @ArthurZucker, @eunseojo may you please review this PR?

### CODE REVIEW

File: chapters/ko/_toctree.yml [hunk 1]
```diff
@@ -26,4 +26,23 @@
...
     title: 단원 마무리 퀴즈
     quiz: 1
-    
\ No newline at end of file
+
+- title: 2. 🤗 Transformers 사용하기
+  sections:
+  - local: chapter2/1
+    title: 단원 소개
+  # - local: chapter2/2
```
Comment (lewtun) [inline chapters/ko/_toctree.yml:34]: Could we please remove the commented out sections to avoid merge conflicts with other contributors?

**Review by lewtun — COMMENTED** [review]
Thanks for translating this section @wonhyeongseo and welcome to the 🤗 community!

We just need to remove some commented out parts of the table of contents before we merge, but otherwise this looks great!

File: chapters/ko/_toctree.yml (hunk 1 above)
Comment (wonhyeongseo) [reply chapters/ko/_toctree.yml]: Nice to meet you, @lewtun! I removed the comments. Happy to contribute :)

**Comment by lewtun** [comment]
> * Package names and their everyday term counterpart is quite confusing to differentiate. Is there a way to tell easily? (🤗 transformers and just the model, 🤗 tokenizers and just regular tokenizers etc.)

Indeed, this is something we ran into while write [our book](https://transformersbook.com/) :)

The approach we took there was:

* Use 🤗 {library name} for the Hugging Face libraries
* Use lower case for the generic reference like "transformers" or "tokenizers"



> * I also got to know a fellowship for huggingface. How do I apply? Do I need a nomination from a prior fellow?

There isn't an application process and we reach out to people in the community who are doing great work (and might some support from the 🤗 team). The best way to become a fellow is to build cool things with the HF ecosystem and/or help the community!



> * I checked the previous assignee's repo before this PR. What is the point of no return? 3 months since the last commit? When can another person takeover? (still very new to opensource, sorry for the novice question)

Good question! I think we can ask on the issue if anyone is still actively working on their sections. If not, you're welcome to tackle them!

**Review by lewtun — APPROVED** [review]
Thanks for the fix, it looks great now!

**Pull request merged**